Makefile

```diff
@@ -24,6 +24,7 @@ REPOPATH ?= $(ORG)/minishift
 ifeq ($(GOOS),windows)
 	IS_EXE := .exe
 endif
+MINISHIFT_BINARY ?= $(GOPATH)/src/$(REPOPATH)/out/$(GOOS)-$(GOARCH)/minishift$(IS_EXE)
```

Comment:
@hferentschik original path is: `$(BUILD_DIR)/$(GOOS)-$(GOARCH)/minishift$(IS_EXE)`, which is equal on linux to `./out/linux-amd64/minishift` and it starts from make file folder. When I tried to make it absolute path, I have added `$(GOPATH)/src/$(REPOPATH)` = `/home/agajdosi/golang/src/github.com/minishift/minishift`.